src/logger/AdvancedConsoleLogger.ts

```diff
@@ -46,16 +46,16 @@ export class AdvancedConsoleLogger implements Logger {
      */
     logQuerySlow(time: number, query: string, parameters?: any[], queryRunner?: QueryRunner) {
         const sql = query + (parameters && parameters.length ? " -- PARAMETERS: " + this.stringifyParams(parameters) : "");
-        console.log(chalk.underline.yellow(`query is slow:`) + " " + chalk.bold(PlatformTools.highlightSql(sql)));
-        console.log(chalk.underline.yellow(`execution time:`), time);
+        PlatformTools.logWarn(`query is slow:`, PlatformTools.highlightSql(sql, true));
```

Comment:
told on sql does not work (it highlights only first word of sql query), so you can remove it.

Reply:
just what I did yesterday with `query failed` error message - I removed bold from there